doc-Managing_Infrastructure_and_Inventory/topics/Performing_SmartState_Analysis_on_Hosts.adoc

```diff
@@ -1,7 +1,10 @@
 = Performing SmartState Analysis on Hosts
 
 Perform a SmartState Analysis on a host to collect additional information about it, such as patches, CPU, and memory.
-
+[NOTE]
+====
+When VMware ESX or ESXi hypervisors are present please consider the following: *ESX* hosts utilize a service console for host management and can be accessed using SSH. *ESXi* hosts lack a service console and therefore SSH cannot be used to obtain information sets for patches, services, Linux packages, user groups, SSH Config, and FS Files. 
+====
 [NOTE]
 ====
```

Comment:
Instead of back-to-back note boxes I'd combine them into one containing two bullet points.